lib/intel.py

```diff
@@ -60,13 +60,19 @@ def run(self):
                     for domain in homenet.bad_domains[threat]:
                         self.all_domains.add(domain)
 
-                # Adding user blacklisted IP addresses
-                for ip in homenet.blacklist:
-                    homenet.bad_ips['User'].append(ip)
-                    self.all_ips.add(ip)
+                # Adding user blacklisted IP addresses and domains
+                utils.flush_ipset_list('blacklist-user')
+                for entry in homenet.user_blacklist:
```

Comment:
As mentioned above, create a new list inside the Network object. 
Something like: user_domain_blacklist